miso-web/src/main/webapp/scripts/lims.js

```diff
@@ -1098,3 +1098,16 @@ Utils.sorting = {
         || a.alias.localeCompare(b.alias);
   }
 };
+
+Utils.notes = {
+   addNote: function(entityType, entityId, internalOnly, text) {
+        Utils.ajaxWithDialog('Adding Note', 'POST', window.location.origin + '/miso/rest/note/' + entityType + '/' + entityId, {
+            internalOnly: internalOnly == 'on',
+            text: text
+          }, function() { Utils.page.pageReload(); }); 
```

Comment:
Can simplify to `Utils.page.pageReload` instead of declaring a new function. Same in `deleteNote` function below